### DISCUSSION TITLE
Would it be possible to add this as a feature - GraphicsCaptureSession.IsBorderRequired Property for Display Capture?

### DISCUSSION THREAD

**Discussion by Dunravin** [obsproject/obs-studio#9590] [Development Questions] [answered]
GraphicsCaptureSession.IsBorderRequired Property

Definition
Namespace:
[Windows.Graphics.Capture](https://learn.microsoft.com/en-us/uwp/api/windows.graphics.capture?view=winrt-20348)
[Edit](https://github.com/MicrosoftDocs/winrt-api/blob/docs//windows.graphics.capture/graphicscapturesession_isborderrequired.md)
Gets or sets a value specifying whether the capture operation requires a colored border around the window or display to indicate that a capture is in progress.

https://learn.microsoft.com/en-us/uwp/api/windows.graphics.capture.graphicscapturesession.isborderrequired?view=winrt-20348#Windows_Graphics_Capture_GraphicsCaptureSession_IsBorderRequired

**Comment by sebastian-s-beckmann** [comment]
As far as I know, it already gets set. See https://github.com/obsproject/obs-studio/blob/839461dc9be78cd6dacf6b9d3a42f911fd6e07a3/libobs-winrt/winrt-capture.cpp#L410 and https://github.com/obsproject/obs-studio/blob/839461dc9be78cd6dacf6b9d3a42f911fd6e07a3/libobs-winrt/winrt-capture.cpp#L490
However, it's only available on Windows 11 (the version mentioned in the docs is a Windows Server version, not normal Windows 10).

**Reply by Dunravin** [reply]
Ah thanks I missed the Windows Server part and cannot run Win 11. So there's no hope for Windows 10?

**Reply by Dunravin** [reply]
I'll try and see if there's any other app setting the property to true because until I reinstalled windows a few weeks ago I never had this yellow border.

**Reply by WizardCM** [reply, marked as answer]
Correct, unless Microsoft changes their mind, the ability for us to turn off the yellow border is limited exclusively to Windows 11. It's quite frustrating as they announced the ability before they announced Windows 11, and then version gated it to Windows 11 only.

**Comment by WizardCM** [comment]
Could you elaborate on why you want this as a property?

**Reply by Dunravin** [reply]
Yes of course. When using Display Capture and the Windows 10 method within a yellow border is placed around the screen. This is undesirable (a quick search of the internet reveals I'm not alone here)

An option in OBS to set this property to true or false would be desirable.

I understand if any other app has set this property to set required to true the border would still be displayed but at least the user could hunt down which other app that was and disable it there too. (Snipping tools etc shipped with windows and 3rd party apps)
